src/modules/common/ChatbotPrompt.tsx

```diff
@@ -49,10 +48,15 @@ const ChatbotPrompt = ({ id }: Props): JSX.Element | null => {
   } = hooks.useAppSettings<ChatbotPromptSettings>({
     name: SettingsKeys.ChatbotPrompt,
   });
+  const chatbotPrompt = chatbotPrompts?.[0];
```

Comment:
I would check the length as well, it might easily fail here.

Reply:
As dumb as it sounds, accessing an element out of the array bounds will simply return `undefined`, not an error, and it does not even throw ... I know... JS what have you done ?